db/migrate/20221102100727_drop_old_ont_tables.rb

```diff
@@ -0,0 +1,11 @@
+class DropOldOntTables < ActiveRecord::Migration[7.0]
```

Comment:
Do we need to drop all of these? ont_runs and ont_flowcells are going to be used? Maybe there is a specific reason?

Reply:
I dropped them to nuke the old data and because I thought the run data would most likely change from the current tables. Happy to keep them though, not sure whats best

Reply:
OK. That's fine. Doesn't matter either way.